src/data/quizzes/index.ts

```diff
@@ -37,4 +37,56 @@ const quizzes: RawQuizzes = {
   },
 }
 
+export const ethereumBasicsQuizzes: QuizzesSection[] = [
+  {
+    id: "what-is-ethereum",
```

Comment:
Not blocking but we need to update this question:

<img width="500" alt="Screenshot 2023-05-29 at 09 10 20" src="https://github.com/ethereum/ethereum-org-website/assets/62268199/1a0f25f4-d8d5-48a2-9060-7b4d82201835">

Reply:
Which part, explanation? options?

Reply:
Sorry, we removed YouTube information from this page. Not actionable by you, I'll sort it! :)

Reply:
How should this be updated ? cc @jmcook1186